### PR TITLE
Enums announcement

### PR DESCRIPTION
**Pull request by LauraDuRy**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by LauraDuRy** [comment]
Need to update the docs link

File: src/routes/blog/post/introducing-enums-sdk-support/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+account.createOAuth2Session(OAuthProvider.Apple, '[LINK_ON_SUCCESS]', '[LINK_ON_FAILURE]', true);
+```
+
+This new feature will improve the experience of building with Appwrite and lower the barrier for new developers to start. As always, we aim to make life easier, not harder, and we look forward to your feedback. 
+
+# Resources
+
+Visit our documentation to learn more about Enums SDK support, join us on Discord to be part of the discussion, view our blog and YouTube channel, or visit our GitHub repository to see our source code.
+
+- [Docs](https://appwrite.io/docs)
```
Comment (adityaoberai) [inline src/routes/blog/post/introducing-enums-sdk-support/+page.markdoc:46]:
```suggestion
- [Docs](https://appwrite.io/docs/sdks#enums)
```

Comment (adityaoberai) [reply src/routes/blog/post/introducing-enums-sdk-support/+page.markdoc:46]: Since the enums section would be available here, we can have this link

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@LauraDuRy mostly looks good, just have one change recommendation

**Review by adityaoberai — APPROVED** [review]

**Pull request merged**